themes/frontierline/content-views/content-mini.php

```diff
@@ -2,18 +2,28 @@
 /**
  * Display a post with just the thumbnail, title, and date.
  */
+
+$post_classes = array(
+  'post-mini',
+);
+
+if (get_theme_mod('frontierline_no_post_thumbnail') === '1') {
+  array_push($post_classes, 'post-thumb-disabled');
+}
 ?>
 
-<div class="post-mini">
+<div <?php post_class($post_classes) ?>>
```

Comment:
This is bringing along the standard classes and getting the padding from the `.single .post` rule on single pages. We can either override that with some higher specificity in the category menu style, or maybe just add the `post-thumb-disabled` class explicitly instead of using `post_class()`?

![post-mini-inherited-padding](https://user-images.githubusercontent.com/205591/36274569-f9016346-123c-11e8-90cf-e65b367586b2.png)

Reply:
@MikkCZ Now we know why this didn't use post_class ;)

Reply:
Oh, I see. Let's revert it back. ;)

Reply:
It _could_ use `post_class()`, we'd just need to override any other styling it inherits, or otherwise use something else as a style hook for the single pages instead of `.post`. 

The content-mini template is sort of a special case anyway, as it's only used in this one context (so far; it could be used elsewhere) so I'd be in favor of _not_ using `post_class()` just to avoid the baggage that comes with it.